src/Adyen/Service/Builder/OpenInvoice.php

```diff
@@ -28,46 +28,43 @@ class OpenInvoice
     /**
      * Build invoice line items for open invoice payment methods
      *
-     * @param string $description
-     * @param float $itemAmount
-     * @param float $itemVatAmount
-     * @param float $itemVatPercentage
-     * @param int $numberOfItems
-     * @param string $vatCategory
      * @param string $itemId
-     * @param string|null $productUrl
+     * @param string $description
+     * @param string $itemCategory
+     * @param int $amountExcludingTax
+     * @param int $amountIncludingTax
+     * @param int $taxAmount
+     * @param int $taxPercentage
+     * @param int $quantity
+     * @param string $productUrl
      * @param string|null $imageUrl
      * @return array
      */
     public function buildOpenInvoiceLineItem(
-        $description,
-        $itemAmount,
-        $itemVatAmount,
-        $itemVatPercentage,
-        $numberOfItems,
-        $vatCategory,
-        $itemId,
-        $productUrl = null,
-        $imageUrl = null
-    ) {
+        string $itemId,
```

Comment:
For a minor version, I believe this would break integrations that are expecting only patch updates, should we move new params to the end of the arg list as non mandatory?